virttest/guest_agent.py

```diff
@@ -725,3 +725,54 @@ def fsthaw(self, check_status=True):
                     raise
             return ret
         return -1
+
+    def openfile(self, args):
+        """
+        Send guest agent command "guest-file-open" to open a file with "w+" mode.
+
+        :params args: It includes file's 'path' and 'mode' option.
+         path = /tmp/agentfile and mode = "w+".
+         if file doesn't exist,mode w+ will create a new file.
+        :return: true if the response received, False if return NULL
+        """
+        cmd = "guest-file-open"
+        if self.check_has_command(cmd):
+            return self.cmd(cmd, args)
+
+    def closefile(self, args):
+        """
+        Send guest agent command "guest-file-close" to close a file with eixst handle value.
+
+        :params args: It includes 'handle' option.
+        handle value will be generated when open file.
+        :return: true if the response received, False if return NULL
```

Comment:
This function has no return.

Reply:
@dinghuiqing @vivianQizhu  I reconstructed this function.  please review it again.

Reply:
(1/2) smp_2.1024m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.myprovider.qemu_guest_agent.virtio_serial.check_guest_file: PASS (418.97 s)
 (2/2) smp_2.1024m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.myprovider.qemu_guest_agent.isa_serial.check_guest_file: PASS (421.96 s)